goblet/resources/eventarc.py

```diff
@@ -20,7 +20,7 @@ class EventArc(Handler):
 
     resource_type = "eventarc"
     # Cloudfunctions gen 2 is also supported
-    valid_backends = ["cloudrun"]
+    valid_backends = ["cloudrun", "cloudfunction"]
```

Comment:
this is just cloudfunctionv2 though right?

Reply:
Yeah oops. I've been trying to keep it possible to specify 2nd gen cloudfunctions with "cloudfunction" as backend and the version set in client_versions, but it's too much of a headache so I'll just change it to where the only way to specify cloudfunctions 2nd gen is with `backend="cloudfunctionv2"`.